java/jdbc/src/main/java/io/vitess/jdbc/VitessPreparedStatement.java

```diff
@@ -141,18 +136,17 @@ public ResultSet executeQuery() throws SQLException {
                             .streamExecute(context, this.sql, this.bindVariables, tabletType, vitessConnection.getIncludedFields());
```

Comment:
`show` statements do not work with streamExecute, and it's possible to set streaming type connection-wide in ConnectionProperties. So I think we either need to still handle `show` statements, or ask @sougou to support them with streamExecute.

I ran into this earlier, and it will fail if anyone enables streaming because when the VitessDriver starts up it executes `SHOW VARIABLES LIKE...` which will immediately fail.